api/types/registry/registry.go

```diff
@@ -102,3 +105,14 @@ type SearchResults struct {
 	// Results is a slice containing the actual results for the search
 	Results []SearchResult `json:"results"`
 }
+
+// ManifestInspect describes the result obtained from contacting the
+// registry to retrieve image metadata
+// TODO(nishanttotla): Decide which other fields to add here
+type ManifestInspect struct {
+	// Digest is the content addressable digest for the image on the registry
+	Digest digest.Digest
```

Comment:
@aaronlehmann I'm inclined towards making it as general as possible. So returning the top-level manifest (and config blob) sounds like a better idea, especially because then the client can deal with it as it likes.